services/web-app/components/svg-libraries/icon-library/icon-library.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import metaData from '@carbon/icons/metadata.json'
+import { Column, Grid } from '@carbon/react'
+import { debounce, groupBy } from 'lodash-es'
```

Comment:
I'm surprised that the `lodash` dep we already have doesn't fulfill this requirement. What error were you seeing when trying to pull these in via regular `lodash`? It'd be best to use one or the other of these libs instead of both.

Reply:
Agree with Joe, saw this on a different component I migrated, if you just change it to `lodash` I think it will work.